editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -1671,43 +1671,43 @@ void SpatialEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 			}
 		}
 		if (ED_IS_SHORTCUT("spatial_editor/bottom_view", p_event)) {
-			cursor.y_rot = 0;
+			cursor.y_rot = 2.0 * Math_PI * Math::round(cursor.y_rot / (2.0 * Math_PI));
```

Comment:
There's a math method to do exactly that: `Math::stepify(cursor.y_rot, 2.0 * Math_PI)`.